searx/data/locales.json

```diff
@@ -62,8 +62,8 @@
     "zh-Hant-TW": "中文, 台灣 (Chinese, Taiwan)"
   },
   "RTL_LOCALES": [
+    "he",
     "fa-IR",
-    "ar",
-    "he"
+    "ar"
   ]
```

Comment:
List needs to be sorted to avoid patches like this one / see:

- https://github.com/searxng/searxng/pull/3233